js/ui/grid_core/ui.grid_core.columns_view.js

```diff
@@ -821,38 +812,32 @@ exports.ColumnsView = modules.View.inherit(columnStateMixin).inherit({
     },
 
     scrollTo: function(pos) {
-        const that = this;
-        const $element = that.element();
-        const $scrollContainer = $element && $element.children('.' + that.addWidgetPrefix(SCROLL_CONTAINER_CLASS)).not('.' + that.addWidgetPrefix(CONTENT_FIXED_CLASS));
-
-        that._skipScrollChanged = false;
+        const $element = this.element();
+        const $scrollContainer = $element && $element.children('.' + this.addWidgetPrefix(SCROLL_CONTAINER_CLASS)).not('.' + this.addWidgetPrefix(CONTENT_FIXED_CLASS));
 
-        if(typeUtils.isDefined(pos) && typeUtils.isDefined(pos.left) && that._scrollLeft !== pos.left) {
-            that._scrollLeft = pos.left;
+        if(typeUtils.isDefined(pos) && typeUtils.isDefined(pos.left)) {
```

Comment:
Why is the `that._scrollLeft !== pos.left` condition removed?

Reply:
Returned